### ISSUE TITLE
pandas 1.x?

### ISSUE THREAD

**Issue by WillForan** [shlomikushchi/zipline-trader#141]
Is support for pandas 1.x in the scope of zipline-trader? Is this up to  quantopian/zipline? (https://github.com/quantopian/zipline/issues/2243 https://github.com/quantopian/zipline/issues/2616)

I started to see what the upgrading looks like 
https://github.com/WillForan/zipline-trader/commits/pandas_update 

```
nosetests -e zipline.utils.numpy_utils
...
FAILED (SKIP=6, errors=333, failures=66)
```

eek

**Comment by shlomikushchi** [comment]
Hi, not currently no. once everything is stable we could start looking at upgrades.

**Comment by RiseT** [comment]
For what it's worth, shortly before Quantopian announced their shutdown, one of zipline's devs had started a [branch ](https://github.com/quantopian/zipline/commits/new-new-new) with the obvious (well, at least to me, an aspiring Python noob...) intention of making zipline compatible with the latest pandas versions. I guess this will never get completed now, but hopefully it's a decent starting point.

**Comment by shlomikushchi** [comment]
Hi, @WillForan did you happen to make this transition? 
I would like to understand if this is possible and what is the effort required to move to pandas 1.X

**Comment by atarax** [comment]
@WillForan i started working on this, and except for quarter-estimates all tests are passing. https://github.com/shlomikushchi/zipline-trader/pull/165

**Comment by shlomikushchi** [comment]
merged in 1.6.0
